stubs/PyYAML/yaml/emitter.pyi

```diff
@@ -35,7 +35,7 @@ class Emitter:
     line: Any
     column: Any
     whitespace: Any
-    indention: Any
+    indentation: Any
```

Comment:
This is what it's actually called in pyyaml (at least as of version 6.0), so we can't change this. Same for the other change in this file.